engine/src/main/java/org/terasology/config/SelectModulesConfig.java

```diff
@@ -25,4 +32,60 @@ public boolean isChecked() {
     public void setIsChecked(boolean value) {
         this.isChecked = value;
     }
+
+    public boolean isLibraryChecked() {
+        return isLibraryChecked;
+    }
+
+    public void setIsLibraryChecked(boolean value) {
+        this.isLibraryChecked =value;
```

Comment:
I think there should be a space after the `=`, here and on the other setters (not sure if it's reported by Checkstyle but I always saw spaces both before and after operators in the Terasology codebase)